android/src/main/java/com/inbrain/InBrainSurveysModule.java

```diff
@@ -92,12 +93,29 @@ public void setDataOptions(final ReadableMap data) {
         InBrain.getInstance().setDataOptions(toHashMap(data));
     }
 
+
+    // ************************x
+    // ***** OPEN WALL ******
     // ************************
-    // ***** SHOW SURVEYS *****
-    // ************************
+
+    // TODO: - Move the initFromRaw to Android SDK
```

Comment:
You can do this `TODO` right now with 2.5.0.

Reply:
done